anko/idea-plugin/xml-converter/test/org/jetbrains/kotlin/android/xmlconverter/BaseXmlConverterTest.java

```diff
@@ -1,22 +1,23 @@
 package org.jetbrains.kotlin.android.xmlconverter;
 
+import static kotlin.collections.SetsKt.setOf;
+import static kotlin.io.FilesKt.readText;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+
 import kotlin.text.Charsets;
 import org.junit.Rule;
 import org.junit.rules.TestName;
-import java.io.File;
-
-import static kotlin.collections.SetsKt.*;
-import static kotlin.io.FilesKt.*;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
 
 public class BaseXmlConverterTest {
 
     @Rule
     public TestName name = new TestName();
 
     protected void doLayoutTest() {
-        File testDataDir = new File("xml-converter/testData");
+        File testDataDir = new File("testData");
```

Comment:
See my comment above; the test was failing because the abstract pathname was set incorrectly. This was looking at `/code/github/anko/anko/idea-plugin/xml-converter/xml-converter/testData`, when it should be `/code/github/anko/anko/idea-plugin/xml-converter/testData`. 

Again, if I'm incorrect with these path changes, please let me know, because I'm not too certain how the abstract pathname gets set.